src/app/level-navigate/level-navigate.component.html

```diff
@@ -0,0 +1,9 @@
+<div id="level-navigate" *ngIf="levelsURL.length > 1">
+  <div *ngFor="let levelURL of levelsURL; index as i;">
+    <a [href]="levelURL" [ngClass]="{'active-level': (i+1) === userCurrentLevel }">
```

Comment:
can the comparison operation be moved to the ts file?

Reply:
To bind a different class to specific HTML element, we would need to do a comparison inline, this way. There may not be a .ts way for this.